examples/gno.land/r/gnoland/pages/admin.gno

```diff
@@ -0,0 +1,86 @@
+package gnopages
+
+import (
+	"std"
+	"strings"
+
+	"gno.land/p/demo/avl"
+)
+
+var (
+	adminAddr     std.Address
+	moderatorList avl.Tree
+	inPause       bool
+)
+
+func init() {
+	// adminAddr = std.GetOrigCaller() // FIXME: find a way to use this from the main's genesis.
```

Comment:
shall we open an issue for this?

Reply:
Just check, I think we already have one, it’s related to the thing we said about having a way to skip the need of faucet for local development, where we could imagine having a detection of accounts on the FS to automatically craft a genesis.

Note that we already have a system which patches the genesis to have a dynamic hostname.

Reply:
Can't we set this value in the `init` method (constructor)?

Reply:
We can, but not with « auto-genesis »; we need to have the auto-genesis taking an account as argument to simulate the initial uploaded.